macro/Version.py

```diff
@@ -47,10 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'jade':
+        if version.lower() == 'kame':
```

Comment:
Uh, not sure, I didn't really think about it, but I guess that's right. Is that what you want it to be?